projects/storefrontapp-e2e-cypress/cypress/integration/regression/product-configuration.e2e-spec.ts

```diff
@@ -0,0 +1,429 @@
+import * as cart from '../../helpers/cart';
+import * as configuration from '../../helpers/product-configuration';
+import * as configurationOverview from '../../helpers/product-configuration-overview';
+import * as productSearch from '../../helpers/product-search';
+
+const electronicsShop = 'electronics-spa';
+const testProduct = 'CONF_CAMERA_SL';
+const testProductMultiLevel = 'CONF_HOME_THEATER_ML';
+
+// UI types
+const radioGroup = 'radioGroup';
+const single_selection_image = 'single_selection_image';
+const checkBoxList = 'checkBoxList';
+
+// Group Status
+const ERROR = 'ERROR';
+const COMPLETE = 'COMPLETE';
+const WARNING = 'WARNING';
+
+// List of groups
+const BASICS = 'Basics';
+const SPECIFICATION = 'Specification';
+const DISPLAY = 'Display';
+const LENS = 'Lens';
+const OPTIONS = 'Options';
+const GENERAL = 'General';
+const VIDEO_SYSTEM = 'Video System';
+const AUDIO_SYSTEM = 'Audio System';
+const SOURCE_COMPONENTS = 'Source Components';
+const PROJECTOR = 'Projector';
+const PROJECTOR_SCREEN = 'Projection Screen';
+const FRONT_SPEAKERS = 'Front Speakers';
+const CENTER_SPEAKER = 'Center Speaker';
+const REAR_SPEAKER = 'Rear Speakers';
+const SUBWOOFER = 'Subwoofer';
+const FLAT_PANEL = 'Flat-panel TV';
+
+// List of attributes
+const COLOUR_HT = 'COLOUR_HT';
+const CAMERA_PIXELS = 'CAMERA_PIXELS';
+const CAMERA_DISPLAY = 'CAMERA_DISPLAY';
+const CAMERA_MODE = 'CAMERA_MODE';
+const CAMERA_SD_CARD = 'CAMERA_SD_CARD';
+const ROOM_SIZE = 'ROOM_SIZE';
+const CAMERA_FORMAT_PICTURES = 'CAMERA_FORMAT_PICTURES';
+const PROJECTOR_TYPE = 'PROJECTOR_TYPE';
+const GAMING_CONSOLE = 'GAMING_CONSOLE';
+const SPEAKER_TYPE_FRONT = 'SPEAKER_TYPE_FRONT';
+
+// List of attribute values
+const WHITE = 'COLOUR_HT_WHITE';
+const TITAN = 'COLOUR_HT_TITAN';
+const SDHC = 'SDHC';
+const JPEG = 'JPEG';
+const PROJECTOR_LCD = 'PROJECTOR_LCD';
+const P5 = 'P5';
+const GAMING_CONSOLE_YES = 'GAMING_CONSOLE_YES';
+const GAMING_CONSOLE_NO = 'GAMING_CONSOLE_NO';
+
+// List of conflict groups
+const CONFLICT_FOR_GAMING_CONSOLE = 'Conflict for Gaming Console';
+
+// Conflict message
+const Conflict_msg_gaming_console =
+  'Gaming console cannot be selected with LCD projector';
+
+context('Product Configuration', () => {
+  beforeEach(() => {
+    cy.visit('/');
```

Comment:
should login first.

Reply:
get this error:
![Screen Shot 2021-03-30 at 5 55 37 PM](https://user-images.githubusercontent.com/44440575/113055624-2dcab480-9181-11eb-91df-3a2e7d4c7b9d.png)

Reply:
Also this test is for a different configurator (variant configurator). Supposed to be run in b2c, no login needed